packages/expo-video/ios/ContentKeyManager.swift

```diff
@@ -0,0 +1,21 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+import AVFoundation
+
+internal class ContentKeyManager {
+  let contentKeySession: AVContentKeySession
+  let contentKeyDelegate: ContentKeyDelegate
+  let contentKeyDelegateQueue = DispatchQueue(label: "\(Bundle.main.bundleIdentifier).ExpoVideo.ContentKeyDelegateQueue")
```

Comment:
We don't need to use the bundle identifier, this 👇 should be fine and easy to find in the threads stacktraces.
Also, having one global queue for all content key managers might be a better option – spawning too many queues is not good.

```suggestion
  static let contentKeyDelegateQueue = DispatchQueue(label: "dev.expo.video.ContentKeyDelegateQueue")
```